examples/react-solana/src/App.tsx

```diff
@@ -63,7 +62,6 @@ createWeb3Modal({
   themeMode: 'light',
   chains,
   wallets: [
-    new BackpackWalletAdapter(),
```

Comment:
This should be handled already on `main` because the BackpackWalletAdapter should be already exported from `@solana/wallet-adapter-wallets` package now, I'll double check.